### PR TITLE
Added Audio as supported content type

### PR DESCRIPTION
**Pull request by bulicamihai**

### CODE REVIEW

**Comment by bulicamihai** [comment]
[Notifications push](https://trello.com/c/0LA18cLX/179-notifications)

**Comment by MihaiSorinMoisa** [comment]
Whitelisting needs to be implemented as well. Don't forget to update the readme.

**Comment by MihaiSorinMoisa** [comment]
`readme.md:67` Update the readme to specify the fact that you can filter the notifications based on `Audio` as well.

File: helm/notifications-push/app-configs/notifications-push_delivery.yaml [hunk 1]
```diff
@@ -7,5 +7,5 @@ env:
   CONSUMER_BACKOFF: "2"
   NOTIFICATIONS_RESOURCE: "content"
   CONTENT_URI_WHITELIST: "^http://(methode|wordpress-article|content)(-collection|-content-placeholder)?-(mapper|unfolder)(-pr|-iw)?(-uk-.*)?\\.svc\\.ft\\.com(:\\d{2,5})?/(content|complementarycontent)/[\\w-]+.*$"
-  CONTENT_TYPE_WHITELIST: "application/vnd.ft-upp-article+json,application/vnd.ft-upp-content-package+json"
+  CONTENT_TYPE_WHITELIST: "application/vnd.ft-upp-article+json,application/vnd.ft-upp-content-package+json",application/vnd.ft-upp-audio+json"
```
Comment (MihaiSorinMoisa) [inline helm/notifications-push/app-configs/notifications-push_delivery.yaml]: Remove the `"` after `application/vnd.ft-upp-content-package+json`. The whitelist should be `"application/vnd.ft-upp-article+json,application/vnd.ft-upp-content-package+json,application/vnd.ft-upp-audio+json"`

**Pull request merged**